src/Compilers/CSharp/Portable/Parser/Blender.Reader.cs

```diff
@@ -307,6 +304,7 @@ internal static bool IsFabricatedToken(SyntaxKind kind)
                     case SyntaxKind.GreaterThanGreaterThanEqualsToken:
                     case SyntaxKind.GreaterThanGreaterThanGreaterThanToken:
                     case SyntaxKind.GreaterThanGreaterThanGreaterThanEqualsToken:
+                    case SyntaxKind.DotDotToken:
```

Comment:
this is what fixes incremental parsing.  `..` is now always a 'fabricated' (like `<<` is), causing the incremental parser to not reuse those.  Instead, we always just produce single `dot` tokens, and allow the parser to decie what to do with them.